crates/bitwarden/src/crypto.rs

```diff
@@ -383,16 +383,16 @@ impl<T: Encryptable<Output>, Output, Id: Hash + Eq> Encryptable<HashMap<Id, Outp
     }
 }
 
-impl<T: Decryptable<Output>, Output, Id: Hash + Eq> Decryptable<HashMap<Id, Output>>
+impl<T: Decryptable<Output>, Output, Id: Hash + Eq + Copy> Decryptable<HashMap<Id, Output>>
```

Comment:
🤔 should this be `Clone`? `Copy` is good enough for the `Uuid` we use today, but is there a good reason it _has_ to be?

Reply:
🤦 got those backward